framework/src/st_things/st_things_request_handler.c

```diff
@@ -638,7 +639,7 @@ static bool add_property_in_post_req_msg(st_things_set_request_message_s *req_ms
 		ST_LOG_V(ST_ERROR, "Invalid property type (%d).", prop->type);
 		break;
 	}
-
+	free(value);
```

Comment:
in 'OCRepPayloadSetPropObjectArrayAsOwner', the value of 'value' will be used in payload.
It will free in other place